### DISCUSSION TITLE
I'm not receiving email notifications on my PR comments

### DISCUSSION THREAD

**Discussion by 1alexvash** [community/community#44376] [Actions]
### Select Topic Area

Product Feedback

### Body

The pull request is [here](https://github.com/1alexvash/AlexCodeBlog/pull/56)

My notifications, settings are like this:
![image](https://user-images.githubusercontent.com/46556479/212530006-72875250-2ed6-4aae-af82-431561c7e24b.png)
But, I don't get any emails on it, how can I collaborate with my team then?

**Comment by radar-blip** [comment]
Maybe it would help to _check_ ✔️  email for participating too.

![parts](https://user-images.githubusercontent.com/39196767/212531369-95e55324-d29e-4323-8bdc-b6ef54775b5c.png)

**Reply by 1alexvash** [reply]
![image](https://user-images.githubusercontent.com/46556479/212533018-addd3b88-cc02-481b-af87-a00ce03870ac.png)
It checked, as well ✅

**Comment by bzkarim250** [comment]
Here are steps you may try:

1.Check that your email address is verified in your GitHub account settings.

2.Check that your notification settings for "Participating" are set to "Watching" or "Watching and Participating" in your GitHub account settings.

3.Check that you have "Watch" selected for the repository in question.

4.Check that the repository owner has enabled email notifications for comments on pull requests.

If all the above steps are set correctly and you still have the problem, try unsubscribing and then resubscribing to the repository to see if it resolves the issue.

**Reply by 1alexvash** [reply]
1) It should be, I use it for years
2) ✔
![image](https://user-images.githubusercontent.com/46556479/212533058-d7fd3fcb-7ee2-43b4-8d57-30af20020d8e.png)
3) ✔
![image](https://user-images.githubusercontent.com/46556479/212533080-89f58fa9-23d4-49cf-b826-9c6cddcd5355.png)
4) Is it 🤔 ?
![image](https://user-images.githubusercontent.com/46556479/212533208-3362aacf-9ade-4c12-bc03-afbc230f2d6d.png)

**Reply by bzkarim250** [reply]
you can reach out to the repository owner and ask them to confirm if email notifications are enabled for pull request comments. it seems to be the good solution.

**Reply by bzkarim250** [reply]
because the repository owner has to enable the email notifications for pull request comments, if they are disabled by default.

**Reply by 1alexvash** [reply]
@bzkarim250 I am a repository owner
![image](https://user-images.githubusercontent.com/46556479/212546562-e25c994c-74e4-4529-b169-eefc2b740494.png)
It's the only thing I've found on repository emails notifications,
is it somehow related to the question at hand?

**Comment by news-manishpatel** [comment]
I am having the same issue. Any reset button? I am not sure what I changed?

**Comment by 1alexvash** [comment]
@news-manishpatel Can you try choosing this option:
![image](https://user-images.githubusercontent.com/46556479/215320687-ab03f766-98c9-4248-8904-907c315491e4.png)
?

I'm not 100% sure, but it feels, like it did fix it for me, so I'd like to get your input as well

**Reply by news-manishpatel** [reply]
Thanks, I will give it go for a couple of days. 

However, I thought if I interact with a PR like approve it or write a comment I would naturally get all activity within that PR. 

![image](https://user-images.githubusercontent.com/91079914/215459273-e9352d6d-5cf8-4af5-b334-b3dca2920e67.png)

**Reply by news-manishpatel** [reply]
After trying all activity for a few hours, I had to go back to participating and mentions. It is too noisy getting emails for PRs I have no interests in. However, yet to receive emails when people reply to your comments.

**Comment by RicFer01** [comment]
I'm having the same problem, did you manage how receive an email after a comment?

**Comment by deleugpn** [comment]
I'm facing this same issue

**Comment by hshoja** [comment]
The same here

**Comment by at-ng** [comment]
I think this issue is posted in the wrong section
> 🚢Actions Discussions
Conversations related to GitHub Actions, the CI/CD platform that allows you to automate your build, test, and deployment pipeline.

Should it not be in the pull request section
https://github.com/orgs/community/discussions/categories/pull-requests

Either way, +1 user not getting email notifications for pull requests (I am getting other notifications).

**Comment by nilsmedina** [comment]
+1 same issue here. Getting all email notifications except replies to my comments.

**Comment by arosha619** [comment]
Make sure your GitHub notification settings are configured to send email notifications for pull request activity. Follow these steps:

Go to your GitHub settings.
Navigate to "Notification center."
Check the settings under "Email notifications" to ensure that "Pull request reviews" and "Pull request comments" are set to send notifications.

**Reply by TraceyC77** [reply]
I came across this issue when searching. I also am getting no notifications for comments on my PRs. I went to the settings as suggested and confirmed that I have checks next to "Reviews, Pushes, Comments" (for Pull Requests), notifications are set to go to GitHub and Email

**Reply by msenyonyi** [reply]
What is notification center?? I don't see anything related to pull request reviews and pull request comments

**Comment by tdbe** [comment]
Keep in mind that getting notifications for anything you are marked as "Participating" in, will STILL result in a lot of spam. Because "Participating" >= "you left a comment on a pr" || "this is your pr"

So as far as I can see, there is no solution to only get notifications for: 1. everything on your pr, and also 2.  once you make a comment on another pr.

The motivation is simple: do not miss when someone chats in a pr you wrote something into. But don't get spammed by all status updates on all PRs or branches you accidentally sneezed at.

**Comment by TraceyC77** [comment]
Since this is reproducible by multiple people, I [opened a support ticket](https://support.github.com/ticket/personal/0/2710930)

**Reply by muum-odoo** [reply]
Can't find the ticket, and I'm facing the same issue

**Comment by signalogic** [comment]
@TraceyC77 thanks for opening the ticket. I have to remind myself every day to check if anyone replied to my comments on issues. It would be so much better to get an e-mail reminder

**Comment by MikeSchulze** [comment]
I have the same issue since a week, it just stopped to get notified.
I checked my email and checked also all notification settings, looks all good but still get nothing notified via email.

Next try, I removed my email address from the emails and added it again. 
I received the email removed mail but never get the verification email, my email worked before without any issues.

That is very frustrating!


[Update]
After I had created a support issue, the support feedback was the notifications are sent.
I changed my private mail address from Vodafone provider to Google and it works.
So looks like Vodafone has an internal junk filter, I do not know.

**Comment by aagranovExtend** [comment]
It's quite random. As a PR author, I normally get every email notification on comments/approvals. Suddenly I'm not getting them all.

**Comment by msenyonyi** [comment]
I have the same issue here. No response at all. I get the notification within Github but no emails related to comments are sent to my account. Really strange. I do get emails when I am asked for a review - or when a PR is merged that I was asked for a review.

I am using a secondary email (not the one linked to my Github account) - as the main notification email address, perhaps that is causing this

**Reply by bernardwiesner** [reply]
I have a very similar issue. For me sometimes I do get notifications on comments on PRs and sometimes not, even on the same PR!

**Reply by qianbinbin** [reply]
This happened to me today, no emails in my inbox and spam box, only in Github notifications.

**Reply by qianbinbin** [reply]
> This happened to me today, no emails in my inbox and spam box, only in Github notifications.

ok, I've received the missing emails, with a delay of 7.5 hours😄

**Comment by Agusioma** [comment]
I had the same issue and it turns out I had to verify my primary email again.  I went to check my emails and weirdly, I saw two bullets:
- Undeliverable 
- Unverified 
I had to click on the verify link under the second bullet (**Unverified**) for notifications to resume. Hope this helps someone.

**Reply by deleugpn** [reply]
I suffer from this problem since 2023. Your reply made me go look and I didn't find anything about re-verifying my emails, but I did find the subscription center, which my account seems to be broken.

I wonder if there's any way to get GitHub support to help

<img width="1151" alt="image" src="https://github.com/user-attachments/assets/089c8b76-ad02-4a0c-8be0-8a370c011833" />

**Reply by Agusioma** [reply]
Interesting @deleugpn. After completing the link request, have things improved?

**Reply by deleugpn** [reply]
the link (and the screenshot) shows that my account is broken 😅

**Comment by ipanasenko** [comment]
Hey!
Right not me and my coworkers are experiencing an issue with not receiving emails abouts PUSHES from others to PRs. Only pushes, other emails are coming just fine.
If you have the same issue - can you upvote, please? https://github.com/orgs/community/discussions/162735

**Comment by DNXie** [comment]
I have this issue for many years. I can only receive email notifications to my old email address.  I did everything suggested above and still couldn't receive email notifications to any other email addresses.

**Comment by msmygit** [comment]
<img width="720" height="600" alt="image" src="https://github.com/user-attachments/assets/6cbc04e4-556a-4731-8020-641548317fa4" />

@1alexvash and others - Make sure if this option on https://github.com/settings/notifications set correctly!
